### PR TITLE
Add South Dakota profile

### PR DESCRIPTION
**Pull request by markbreedlove**

### CODE REVIEW

File: profiles/sd.pjs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    "fetcher_threads": 4,
+    "contributor": {
+        "@id": "http://dp.la/api/contributor/sd",
+        "name": "South Dakota Digital Library"
+    },
+    "sets": "NotSupported",
+    "enrichments_coll": [
+        "/set_context",
+        "/validate_mapv3"
```
Comment (moltude) [inline profiles/sd.pjs]: This enrichment was removed wasn't it?

**Review by moltude — APPROVED** [review]
The validate enrichment on collections could be removed but it isn't a blocker.

File: profiles/sd.pjs (hunk 1 above)
Comment (mdellabitta) [reply profiles/sd.pjs]: yep!

**Comment by markbreedlove** [comment]
I'll submit a separate PR to remove the validate_mapv3 enrichment from all profiles. In this case, I copied the profile from Minnesota, so it needs to be removed there, too; and maybe elsewhere.

I'm removing it from this new South Dakota profile.

Thanks!

**Pull request merged**